webapp/src/components/channel_header_actions/index.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/common';
+
+import {openChannelSettings} from 'actions';
+
+import {isUserConnected, isInstanceInstalled} from 'selectors';
+
+import ChannelHeaderMenuAction from './channel_header_menu';
+
+const mapStateToProps = (state) => {
+    const channelId = getCurrentChannelId(state);
+
+    return {
+        channelId,
+        userConnected: isUserConnected(state),
+        isInstanceInstalled: isInstanceInstalled(state),
+    };
```

Comment:
0/5 to be consistent with how the object is created

```suggestion
    return {
        channelId: getCurrentChannelId(state),
        userConnected: isUserConnected(state),
        isInstanceInstalled: isInstanceInstalled(state),
    };
```